pkg/kn/commands/service/create.go

```diff
@@ -50,6 +50,9 @@ var create_example = `
   # Create a service with port 80
   kn service create s2 --port 80 --image knativesamples/helloworld
 
+  # Create a service with port 80 and port name h2c
+  kn service create s2 --port h2c:80 --image knativesamples/helloworld
```

Comment:
if we've an image which works with h2c port, we can push the example image at `knativesamples`. ping @rhuss

Reply:
Thank you, will check.